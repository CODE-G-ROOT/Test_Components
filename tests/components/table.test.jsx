import { render } from "@testing-library/react"
import Table from '../../src/components/Table'

describe("Captura del componente <Tablla.jsx>", ()=>{
    test("Fotico", ()=> {
        let {container} = render(<Table/>);
        expect(container).toMatchSnapshot();
    });
});