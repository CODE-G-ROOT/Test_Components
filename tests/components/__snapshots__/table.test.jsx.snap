// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Captura del componente <Tablla.jsx> Fotico 1`] = `
<div>
  <div>
    Table
  </div>
</div>
`;
